gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=29.34.3
+version=29.34.4
```

Comment:
Since we introduced a new enum symbol in a public enum, let's bump minor version here.